tests/expected/offset-overflow/main.rs

```diff
@@ -8,13 +8,12 @@ use std::intrinsics::offset;
 
 #[kani::proof]
 fn test_offset_overflow() {
-    let a: [i32; 3] = [1, 2, 3];
-    let ptr: *const i32 = a.as_ptr();
+    let s: &str = "123";
+    let ptr: *const u8 = s.as_ptr();
 
-    // a value that when multiplied by the size of i32 (i.e. 4 bytes)
-    // would overflow `isize`
-    let count: isize = isize::MAX / 4 + 1;
```

Comment:
What's wrong with this?

Reply:
Nothing. I changed this test in #3584 to have an overflow in the multiplication operation, but then realized that it was meant to check overflow in the addition operation, so I reverted the change.